pkl-core/src/main/java/org/pkl/core/stdlib/test/report/TestReport.java

```diff
@@ -24,11 +24,11 @@
 import org.pkl.core.TestResults;
 import org.pkl.core.util.StringBuilderWriter;
 
-public interface TestReport {
+abstract class TestReport {
```

Comment:
Looks like you need to run `./gradlew spotlessApply` to fix formatting across all files. Running `./gradlew check` should show any issues for you locally.

Reply:
Fixed.